lib/beaker/host/unix/exec.rb

```diff
@@ -154,6 +154,9 @@ def mkdir_p dir
   # Recursively remove the path provided
   # @param [String] path The path to remove
   def rm_rf path
+    unless path.start_with?('"', '\'') && path.end_with?('"', '\'')
+      path.gsub!(/(?<!\\) /, '\ ')
```

Comment:
Regular expressions should be commented with what they are attempting to match written in plain English.

Reply:
Added comments for that. If they are not enough, please let me know. Sorry for the long delay.